plugins/ern_v0.14.0+/react-native-reanimated_v1.3.0+/config.json

```diff
@@ -0,0 +1,33 @@
+{
+  "android": {
+    "root": "android",
+    "dependencies": [
+      "androidx.transition:transition:1.1.0"
```

Comment:
@friederbluemle React Native 0.60.x was the first version using Android X,  should this plugin be moved to the `ern_v0.39.0+` folder, or doesn't it work like that?

Reply:
Until now (or recently), there has only been a _very loose coupling_ between version of React Native and ERN. So while there is a default when a new project is created, it should also be possible to use a different combination of RN/ERN for existing projects. There is some internal discussion about consolidating these plugins and removing the need to put them into different folders in the future. Until then I think we should keep the plugin configs at the lowest supported version, and only move them "up" if they specifically contain a config directive that was added in a newer version of ERN. Hope that makes sense.

Reply:
Got it. Ideally, there would be 2 versions of this plugin: One with a dependency on the support library for < RN 0.60. And on with the Android X dependency for >= RN 0.60. 
Anyway, I understand that would make it even more complicated as you will have to introduce plugins per RN version for each ERN version 😅